GET_STARTED.md

```diff
@@ -0,0 +1,235 @@
+availableSensorsInterface.SetProperty($"/{sensorUuid}/unit", "°C")
+```
+
+Unset property:
+
+``` CSharp
+AstarteDevicePropertyInterface availableSensorsInterface =
+        (AstarteDevicePropertyInterface)myDevice
+        .GetInterface(availableSensorsInterfaceName);
+
+availableSensorsInterface.UnsetProperty("/myPath/name");
+```
+
+# Receive data on server owner interface
+
+In Astarte interfaces with server owner interface. Astarte sends data to connected devices. If a device is offline when data is sent, it receives the messages as soon as it reconnects to Astarte.
+
+This section provides instructions on how to receive data on the server’s owner interface using the Astarte Device SDK for C#.
+
+```CSharp
+using AstarteDeviceSDKCSharp.Protocol;
+using AstarteDeviceSDKCSharp.Protocol.AstarteEvents;
+using Newtonsoft.Json;
+
+namespace AstarteDeviceSDKExample
+{
+    public class ExampleGlobalEventListener : AstarteGlobalEventListener
+    {
+
+    }
+}
+```
+
+## Receiving on server owned datastream interface with individual aggregation
+
+``` CSharp
+public override void ValueReceived(AstarteDatastreamEvent e)
+{
+    Console.WriteLine(
+    $"Received datastream value on interface { e.GetInterfaceName()}, " +
+    $"path: { e.GetPath()}, " +
+    $"value:{ e.GetValue()}");
+}
+```
+
+## Receiving on server owned datastream interface with object aggregation
+
+``` CSharp
+public override void ValueReceived(AstarteAggregateDatastreamEvent e)
+{
+    Console.WriteLine(
+    $"Received aggregate datastream object on interface {e.GetInterfaceName()}, " +
+    $"value:{JsonConvert.SerializeObject(e.GetValues())}");
+}
+```
+
+## Receiving set on a server owned properties interface
+
+``` CSharp
+public override void PropertyReceived(AstartePropertyEvent e)
+{
+    Console.WriteLine(
+    $"Received property on interface { e.GetInterfaceName() }, " +
+    $"path: { e.GetPath() }, " +
+    $"value:{ e.GetValue() }");
+}
+```
+
+
+## Receiving unset on a server owned properties interface
+
+``` CSharp
+public override void PropertyUnset(AstartePropertyEvent e)
+{
+    Console.WriteLine(
+    $"Received unset on interface { e.GetInterfaceName() }, " +
+    $"path: { e.GetPath() }");
+}
+```
+
+# Run the example
+
+## Using an unregistered device
+
+### Run the code from the root of the example project with
+
+``` bash
+dotnet run AstarteDeviceSDKExample.csproj --r "<realm>" --p "<pairing-url>" --t "<jwt>"
+```
+
+where `pairing-url` is the URL to reach Pairing API in your Astarte instance, usually `https://api.<your-astarte-domain>/pairing`.
+
+### Using a registered device
+
+### Run the code from the root of the example project with
+
+``` bash
+dotnet run AstarteDeviceSDKExample.csproj --r "<realm>"  --p "<pairing-url>" --d "<device-id>" --c "<credential-secret>"
+```
+
+where `pairing-url` is the URL to reach Pairing API in your Astarte instance, usually `https://api.<your-astarte-domain>/pairing`.
```

Comment:
No need to explain how to run an example in the get started. The get started guide is intended to provide step by step snippets to create your own example.